lms/static/sass/discussion/views/_thread.scss

```diff
@@ -30,11 +29,25 @@ body.discussion, .discussion-module {
 
     .post-header-actions {
       @include float(right);
-      width: flex-grid(3,12);
     }
 
-    .posted-details {
-      color: $gray-d1;
+    .post-body {
+      width: flex-grid(10,12);
```

Comment:
This was already used in this file, so I just followed the same approach. I dug into it a little more and it turns out this is a Bourbon function that is being deprecated, and it doesn't look like it uses flexbox in its implementation. Don't worry, I won't just start using flexbox without some careful prep work!